src/sql/util.py

```diff
@@ -157,6 +157,84 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates
+) -> bool:
+    """
```

Comment:
Please add a docstring explaining every parameter. It's not very clear right now